lib/util/player_configuration.js

```diff
@@ -31,7 +31,10 @@ shaka.util.PlayerConfiguration = class {
   static createDefault() {
     // This is a relatively safe default in the absence of clues from the
     // browser.  For slower connections, the default estimate may be too high.
-    const bandwidthEstimate = 1e6; // 1Mbps
+    // The minus 1kb is needed since the network API is the multiplication of
+    // 25kb, this change helps us distinguish between default value and
+    // API-based value (in AWMA getBandwidthEstimate function)
+    const bandwidthEstimate = 999000; // 1Mbps - 1kb
```

Comment:
If you do what I said above about removing the `getDefaultBandwidth_ `, it should no longer be necessary to change the default bandwidth estimate here.